### ISSUE TITLE
Randomized word spacing and text spacing to make the notes look more orignal

### ISSUE THREAD

**Issue by killbotXD** [saurabhdaware/text-to-handwriting#104]
**Is your feature request related to a problem? Please describe.**
No, it is not related to a problem, but if I have a long assignment, 12-13 pages or more, then the consistency of the handwriting may be a little alarming. So I wanted some way to make it look more natural.

**Describe the solution you'd like**
We can have a range of text spacing and word spacing from which random values would be selected, hence making it a little more natural.

**Comment by rkscodes** [comment]
yeah i also felt the need for same, a bit random is very necessary thing to avoid suspicion, If anyone knows how to make it happen please make a PR or suggest me way to do it, I am willing to work on it
